composables/useBalance.ts

```diff
@@ -4,26 +4,31 @@ import { getKusamaAssetId } from '@/utils/api/bsx/query'
 export default function () {
   const { urlPrefix } = usePrefix()
   const identityStore = useIdentityStore()
-  const balance = computed(() => {
-    switch (urlPrefix.value) {
+
+  const getBalance = (prefix: string, token: string) => {
+    token = token.toLocaleLowerCase()
+    switch (prefix) {
       case 'rmrk':
       case 'ksm':
       case 'ahk':
       case 'ahp':
+      case 'dot':
         return identityStore.getAuthBalance
```

Comment:
It always returns the balance base on the current prefix from the route instead of the prefix you pass here.

Reply:
oh I see getAuthBalance is using urlPrefix

Reply:
Any update?

Reply:
fixed